internal/events/token_pool_created.go

```diff
@@ -65,8 +65,6 @@ func (em *eventManager) confirmPool(ctx context.Context, pool *fftypes.TokenPool
 		return err
```

Comment:
It occurs to me that this operation query is probably not needed at all now. I think it was only ever here to do the resolve.

Reply:
Agreed. Will you remove it in this PR?

Reply:
Yes, will push a quick change to do that.